### DISCUSSION TITLE
Do you have any roadmap?

### DISCUSSION THREAD

**Discussion by Laegel** [ast-grep/ast-grep#295] [Q&A] [answered]
Hey, nice project! I've been working on something similar as a side-project (Rust + Tree Sitter to execute code modification/generation based on AST) but I've not been able to release anything yet. 
Do you have any roadmap? I'm very interested in contributing!

**Comment by HerringtonDarkholme** [comment, marked as answer]
Hi! Thanks for your interest in sat-grep.
Currently all issues and actionable items are tracked by GitHub issues. So you can look at the issue list to see what's on the roadmap of this project.

**Reply by Laegel** [reply]
Awesome, thanks your answer. I've got other questions about implementation details, what led to the creation of this project and other broader questions: where should I ask them?

**Reply by HerringtonDarkholme** [reply]
GH discussion and twitter are both okay to ask these broad questions! I will try to answer them when I am available.(though no guarantee)

**Reply by Laegel** [reply]
Of course, don't worry; thanks for your time! Well, I'm not a Twitter guy so I'll stick to GH Discussions:
- What was the motivation behind this project? 
- Why did you choose Tree Sitter over another technology?
- If other people were to contribute, would you mind moving the issues you've created into the "TODO" column and adding details to them so it's easier for contributors to produce code that matches your expectations?
